src/main/java/com/wanted/notion/domain/dao/PageDao.java

```diff
@@ -0,0 +1,73 @@
+package com.wanted.notion.domain.dao;
+
+import com.wanted.notion.domain.dto.SubPageDto;
+import com.wanted.notion.domain.dto.PageDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class PageDao {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public PageDto findById(Long pageId) {
+        String query = "SELECT * FROM page WHERE pageId = ?";
+        return jdbcTemplate.queryForObject(query, pageDtoMapper, pageId);
+    }
+
+    private RowMapper<PageDto> pageDtoMapper = ((rs, rowNum) ->
+            PageDto.builder()
+                    .pageId(rs.getLong("pageId"))
+                    .title(rs.getString("title"))
+                    .content(rs.getString("content"))
+                    .parentId(rs.getLong("parentId") == 0 ? null : rs.getLong("parentId"))
```

Comment:
```suggestion
                    .parentId(rs.geString("parentId") == null ? null : rs.getLong("parentId"))
```

여기서 getString으로 받아서 null 여부를 체크하는 것이 db를 직관적으로 생각하도록 할 수 있을거 같아요